azavea/pfb-network-connectivity
# Add tile styling project files, and a README

Adds the working files necessary to load and modify the tile styles with
Mapbox Studio Classic, and a README file to explain what's going on.

### Demo

![image](https://user-images.githubusercontent.com/6598836/27806229-57b8f360-6007-11e7-8f09-6c118e299e60.png)

### Notes

The process is a little squirrelly--starting with hand-editing the project files to correct hard-coded absolute file paths and ending with hand-copying the translated styles from the project XML to the style XML we use to do the actual tile generation.  There are probably better ways to go about this, but this does the job.

## Testing Instructions

Follow the README instructions to install Mapbox Studio Classic, modify the file paths, and open the project.  It should show up looking roughly like the screenshot.  Changing a style and saving should result in a diff in the files in `combined_styles.tm2`.

Resolves #506.
